pycba/bridge.py

```diff
@@ -65,6 +65,64 @@ def reverse(self):
 
         """
         self.axle_coords = +self.L - self.axle_coords
+    
+    @classmethod
+    def from_vehicle_sequence(cls, vehicles: list, 
+                              vehicle_spacings: np.ndarray):
```

Comment:
For python 3.8, the type hint needs to be `List[Vehicle]`. `List` may need to be imported from typing if not already.

For later versions of python, `list[Vehicle]` should work, but aiming to support 3.8+ for now.

Reply:
`List[Vehicle]` gives a syntax error since `Vehicle` has not been defined. A similar error is seen when using `-> Vehicle`. Interestingly, there's no error when using `isinstance` within the class method.

Reply:
It's a python thing: https://stackoverflow.com/questions/61544854/from-future-import-annotations